README.md

```diff
@@ -0,0 +1,87 @@
+# uniSimple
+
+<!-- vim-markdown-toc GitLab -->
+
+* [Purpose of this app](#purpose-of-this-app)
+* [About this application](#about-this-application)
+* [Getting Started](#getting-started)
+  * [Running the development server](#running-the-development-server)
+  * [Adding new dependencies](#adding-new-dependencies)
+
+<!-- vim-markdown-toc -->
+
+
+## Purpose of this app
+
+
+The purpose of this app is to provide a user interface for admin users to maintain the 'uniSimple' site.
+
+
+## About this application
+
+
+This application is based on the Wagtail CMS.
+
+## Getting Started
+
+
+### Running the site
+
+You have two options to run the development server:
+
+**1.** In a virtual environment of your choice run the following from the root directory of the project:
```

Comment:
For both examples it is good practice to identify what to install. Prior to using pip, one needs to install Python (3.6.8) and then pip (version 19.0.3). For mac users this will be done by using brew - it's worth documenting this as pre-requisites before running CMS, same goes for docker.